src/cljs/replumb/load.cljs

```diff
@@ -51,6 +60,42 @@
                                   (read-files-and-callback! verbose? (rest file-names) read-file-fn! load-fn-cb))))))
     (load-fn-cb nil)))
 
+(defn read-files-from-cache-and-callback!
+  "Loops over cached-file-names in order to retrieve them. It needs to find
+  both the related .js file and .cache.json file, otherwise keeps looping.
+  If it does not find the cached files calls read-files-and-callback! and
+  tries to load the unevaluated ones."
```

Comment:
Let's add `This function does not check whether parameters are nil, please do it in the caller."` to make it clear ok?